python/whylogs/api/logger/__init__.py

```diff
@@ -20,6 +23,94 @@ def log(
     return TransientLogger(schema=schema).log(obj, pandas=pandas, row=row)
 
 
+def log_classification_metrics(
```

Comment:
Any example of this from end user's perspective?

Reply:
Updated this a bit, but here's two different ways to log model performance metrics with this interface:
1) standalone:
```
import random
import whylogs as why

# some fake example classification data taken from 5 labels
label_set = ['x0', 'x1', 'x2', 'x3', 'x4']
number_of_predictions = 100
example_predictions = [label_set[random.randint(0,4)] for i in range(number_of_predictions)]
# simulate targets being correct more than 85% of the time, then random guess
example_targets = [example_predictions[i] if random.random() > 0.15 else label_set[random.randint(0,4)] for i in range(number_of_predictions)]

# profile result set containing model performance metrics (in an empty DatasetProfile)
model_perf_results = why.log_classification_metrics(targets=example_targets, predictions=example_predictions)

# Because this is a ResultSet, you can write it in the usual ways, e.g.
model_perf_results.writer("whylabs").write()
```

2) Add model performance metrics to an existing DatasetProfile:
```
from whylogs.core.model_performance_metrics import ModelPerformanceMetrics

model_perf = ModelPerformanceMetrics()
model_perf.compute_confusion_matrix(
    predictions=example_predictions,
    targets = example_targets)

first_segment_profile.add_model_performance_metric(model_perf)
```

This is very similar to v0 calling pattern with some minor simplifications.